CHANGELOG.md

```diff
@@ -70,6 +70,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased 2.x]
 ### Added
 - Add GeoTile and GeoHash Grid aggregations on GeoShapes. ([#5589](https://github.com/opensearch-project/OpenSearch/pull/5589))
+- Small bug fix in Search Backpressure's HeapUsageTracker ([#6455](https://github.com/opensearch-project/OpenSearch/pull/6455))
```

Comment:
For small non-customer facing changes , we can skip the changelog.

Reply:
okay, removing it.